modules/ROOT/pages/lb-mapping-rules.adoc

```diff
@@ -8,165 +8,292 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
+|1 | app.example.com | app | /example | http
 |===
 
-This rule maps the subdomain part of your domain name to the application name:
+Alternatively, you can define a pattern to hold the input value:
 
-* `dev.example.com` redirects to `dev-app.cloudhub.io`.
-* `qa.example.com` redirects to `qa-app.cloudhub.io`.
+[%header,cols="10a,20a,20a,20a,10a"]
+|===
+| Rule # | *Input Path* | *Target App* | *Output Path* | *Protocol*
+|1 | example.com/{myPattern} | app-{myPattern} | /data | http
+|===
 
+In this case, both `example.com/bookings` and `example.com/sales` match to `app-bookings/data` and `app-sales/data` respectively, because the variable `_myPattern_` holds these values.
 
-=== 1:1 Mapping
+If the input is:
 
-If you have only one application, you can map the literal app name.
+* `bookings.example.com`, the pattern resolves if the value of `_myPattern_` is `bookings`. 
+* `sales.example.com`, the pattern resolves if the value of `_myPattern_` is `sales`.
 
-[%header,cols="10a,20a,20a,10a"]
+
+=== URL Mapping
+
+You can pass the app name as the *Input Path* and map it directly to the app name in CloudHub:
+
+[%header,cols="10a,20a,20a,20a,10a"]
 |===
-|Rule # |Input URL 2+^| Output URL
-|   |  *URI*  |   *appName* |   *appURI*
-| 0 | /       |    myApp    | /
+| Rule # | *Input Path* | *Target App* | *Output Path* | *Protocol*
+| 1 | /{app}/ | {app} | / | http
 |===
-This maps your default load balancer `lb-demo.lb.anypointdns.net` directly to your app in CloudHub `myApp.cloudhub.io`.
 
-== Indexing the Priority of Rules
+This rule maps requests to `lb-demo.lb.anypointdns.net/{app}`, redirecting them to `{app}.cloudhub.io`.
+The `{app}` pattern matches any application name that you pass in the inbound request.
 
-When creating a mapping rule, assign an index to it to define the rule's priority order.
+=== Host Mapping
 
-A rule defined first, at index `0` has higher priority against other rules defined after it. The higher the index assigned, the less priority the mapping rule has.
+If your SSL endpoint sets a https://en.wikipedia.org/wiki/Wildcard_certificate[wildcard certificate] (such as `*.example.com`), you can use the `subdomain` variable to map any subdomain:
 
-Every rule must have a priority defined. When creating rules, pay attention to the order for each rule to avoid having multiple rules override each other.
+[%header,cols="10a,20a,20a,20a,10a"]
+|===
+| Rule # | *Input Path* | *Target App* | *Output Path* | *Protocol*
+| 1      | /            | {subdomain}  | /             | https
+|===
+
+This rule automatically maps any request passed to a subdomain of `example.com` to the corresponding `appName`:
+
+* Passing `api.example.com` redirects to `api.cloudhub.io`.
+* Passing `application.example.com` maps to `application.cloudhub.io`.
 
-=== Ordering and Prioritizing Rules
+The same behavior applies to the https://en.wikipedia.org/wiki/Subject_Alternative_Name[Subject Alternative Names] (SANs) of your SSL endpoints.
```

Comment:
I suggest rewording to make clear to what "the same behavior" applies, to make this subtopic independent and therefore reusable.